src/main/java/uk/gov/justice/digital/hmpps/cmd/api/domain/NotificationDescription.kt

```diff
@@ -31,11 +30,8 @@ class NotificationDescription {
                 3 -> "rd"
                 else -> "th"
             }
-            val year = if (shiftDate.year != LocalDate.now(clock).year) {
```

Comment:
I removed the year stuff, because it needed the clock passing it, its much cleaner not to and no real loss.